test/webaudio/create_webaudio.c

```diff
@@ -23,9 +23,7 @@ int main()
 		startButton.onclick = () => {
 			if (audioContext.state != 'running') {
 				audioContext.resume();
-#ifdef REPORT_RESULT
-				__ReportResult(0);
-#endif
+				reportResultToServer(0);
```

Comment:
Will this not report more than before? Or was `REPORT_RESULT` always defined?

Reply:
I don't think anyone is ever going to want to build this test without browser reporting included